app/models/concerns/foreman_openscap/host_extensions.rb

```diff
@@ -10,13 +10,13 @@ module HostExtensions
       has_many :arf_reports, :class_name => '::ForemanOpenscap::ArfReport', :foreign_key => :host_id
       has_one :compliance_status_object, :class_name => '::ForemanOpenscap::ComplianceStatus', :foreign_key => 'host_id'
 
-      scoped_search :in => :policies, :on => :name, :complete_value => true, :rename => :compliance_policy,
-                    :only_explicit => true, :operators => ['= ', '!= '], :ext_method => :search_by_policy_name
+      scoped_search :relation => :policies, :on => :name, :complete_value => true, :rename => :compliance_policy,
+                    :only_explicit => true, :operators => ['= '], :ext_method => :search_by_policy_name
 
-      scoped_search :in => :policies, :on => :name, :complete_value => true, :rename => :compliance_report_missing_for,
+      scoped_search :relation => :policies, :on => :name, :complete_value => true, :rename => :compliance_report_missing_for,
                     :only_explicit => true, :operators => ['= ', '!= '], :ext_method => :search_by_missing_arf
```

Comment:
I don't know, I haven't tested whether it works correctly, I'm just sure the first one does not